packages/fx-core/src/plugins/resource/bot/resources/messages.ts

```diff
@@ -1,137 +1,135 @@
-    `'${something}' is not found.`;
+    `'${something}' ${getLocalizedString("core.boot.SomethingNotFound")}`;
   public static readonly SomethingIsNotExisting = (something: string): string =>
-    `'${something}' is not existing.`;
-  public static readonly SomethingIsInWrongFormat = (something: string): string =>
-    `'${something}' is in wrong format.`;
+    `'${something}' ${getLocalizedString("core.boot.SomethingNotExisting")}`;
   public static readonly FailToCreateSomeClient = (clientName: string): string =>
-    `Failed to create '${clientName}'.`;
+    `${getLocalizedString("core.boot.FailedToCreate")} '${clientName}'.`;
   public static readonly FailToProvisionSomeResource = (resource: string): string =>
-    `Failed to provision '${resource}'.`;
+    `${getLocalizedString("core.boot.FailedToProvision")} '${resource}'.`;
   public static readonly FailToUpdateConfigs = (something: string): string =>
-    `Failed to update configs for '${something}'.`;
-  public static readonly FailToListPublishingCredentials = "Failed to list publishing credentials.";
-  public static readonly FailToDoZipDeploy = "Failed to deploy zip file.";
+    `${getLocalizedString("core.boot.FailedToUpdateConfigs")} '${something}'.`;
+  public static readonly FailToListPublishingCredentials = getLocalizedString(
+    "core.boot.FailedListPublishingCredentials"
+  );
+  public static readonly FailToDoZipDeploy = getLocalizedString("core.boot.FailedDeployZipFile");
   public static readonly FailToUpdateMessageEndpoint = (endpoint: string): string =>
-    `Failed to update message endpoint with '${endpoint}'.`;
+    `${getLocalizedString("core.boot.FailedUpdateMessageEndpoint")} '${endpoint}'.`;
   public static readonly FailToDownloadFrom = (url: string): string =>
-    `Failed to download from '${url}'.`;
-  public static readonly FailToFindSomethingFor = (something: string, forsth: string): string =>
-    `Failed to retrieve '${something}' for '${forsth}'.`;
-  public static readonly ClickHelpButtonForDetails =
-    "Please click 'Get Help' button for more details.";
-  public static readonly ClickIssueButtonToReportIssue =
-    "Please click 'Report Issue' button to report the issue.";
+    `${getLocalizedString("core.boot.DownloadFail")} '${url}'.`;
+  public static readonly ClickHelpButtonForDetails = getLocalizedString("core.boot.ClickGetHelp");
   public static readonly CommandExecutionFailed = (command: string): string =>
-    `Failed to run '${command}'.`;
+    `${getLocalizedString("core.boot.FailToRun")} '${command}'.`;
   public static readonly DoSthBeforeSth = (sth: string, beforeSth: string): string =>
-    `Perform command '${sth}' before '${beforeSth}'.`;
-  public static readonly FailToCallAppStudioForCheckingAADApp =
-    "Failed to call App Studio's api to check AAD application's existence.";
+    `${getLocalizedString("core.boot.PerformCommand")} '${sth}' ${getLocalizedString(
```

Comment:
Maybe it is cleaner to use `%s` because `getLocalizedString()` accepts a `params` parameter (like `printf` in c). 
https://github.com/OfficeDev/TeamsFx/blob/a0a04e3e2e78aee74294e089024edd71b1c4d5f3/packages/fx-core/src/common/localizeUtils.ts#L10
https://github.com/OfficeDev/TeamsFx/blob/a0a04e3e2e78aee74294e089024edd71b1c4d5f3/packages/fx-core/resource/package.nls.json#L124

Reply:
fixed, many thx